.github/workflows/docker-image.yml

```diff
@@ -24,11 +24,9 @@ jobs:
 
     - name: Set up Docker Buildx
       id: buildx
-      uses: crazy-max/ghaction-docker-buildx@v2
+      uses: crazy-max/ghaction-docker-buildx@v2.0.0
```

Comment:
Th commit message says that this was updated to v3, but the change here only adds minor and patch numbers.

Reply:
The commit message is updated, but I forgot to update the PR message. I will do now.